src/poetry/console/commands/install.py

```diff
@@ -201,6 +201,8 @@ def handle(self) -> int:
                 "If you want to use Poetry only for dependency management"
                 " but not for packaging, you can disable package mode by setting"
                 " <c1>package-mode = false</> in your pyproject.toml file.\n"
+                "If you did intend to install the current project please make sure"
+                " you set the packages setting in your pyproject.toml file.\n"
```

Comment:
```suggestion
                "If you did intend to install the current project you may need"
                " to set `packages` in your pyproject.toml file.\n"
```

I agree it makes sense to provide additional information. I would only rephrase it slightly because setting `packages` is only required if your package name does not match your project name.